app.yaml

```diff
@@ -0,0 +1,21 @@
+runtime: nodejs10
+ 
+handlers:
+- url: /
+  static_files: web/src/index.html
```

Comment:
Running `npm run build` in the `web/` dir will create a `dist/` dir. That's the content we want to use.